client/platform/desktop/frontend/components/Recent.vue

```diff
@@ -111,29 +123,90 @@ export default defineComponent({
       return 'mdi-image-multiple';
     }
 
+    async function preloadCheck(recent: JsonMetaCache) {
+      //Attempts to preload the data to see if there are any isues
+      try {
+        await api.checkDataset(recent.id);
+      } catch (e) {
+        const result = await prompt({
+          title: 'Error Loading Data',
+          text: e,
+          confirm: true,
+          positiveButton: 'Delete',
+          negativeButton: 'Cancel',
+        });
+        if (result) {
+          try {
+            await api.deleteDataset(recent.id);
+            //Now we need to update recents by removing the dataset from localStorage
+          } catch (err) {
+            snackbar.value = true;
+            errorText.value = err.message;
+          }
+          removeRecents(recent.id);
+        }
+        return;
+      }
+      root.$router.push({ name: 'viewer', params: { id: recent.id } });
+    }
+
+    const headers: DataTableHeader[] = [
+      {
+        text: 'Type',
+        value: 'type',
+        sortable: false,
+        width: 40,
+      },
+      {
+        text: 'Name',
+        value: 'name',
+        sortable: true,
+      },
+      {
+        text: 'Accessed',
+        value: 'accessedAt',
+        sortable: true,
+      },
+      {
+        text: 'Delete',
+        value: 'delete',
+        sortable: false,
+      },
+    ];
+    const toDisplayString = (dateString: string) => new Date(dateString).toLocaleString('en-US',
+      {
+        hour12: false,
+        month: '2-digit',
+        day: '2-digit',
+        year: '2-digit',
+        hour: '2-digit',
+        minute: '2-digit',
+      });
```

Comment:
we already have the `moment.js` dependency, might as well use it here.  https://momentjs.com/docs/#/displaying/

I think you end up with much nicer control over the output string.

`moment(dateString).format('MM DD YY, H:mm')` or something like that.